test/scripts/prune_namespaces.go

```diff
@@ -19,6 +19,11 @@ import (
 )
 
 func main() {
+	// Create kube client
+	if err := framework.InitKubeClient(); err != nil {
+		panic(err)
+	}
+
```

Comment:
Was there any issue with that?

Reply:
Yes, uninitialised client was throwing nill dereference error, not sure why or what changed. Maybe a result of recent operator refactoring.